src/The_Cartridge_Header.md

```diff
@@ -1,65 +1,75 @@
 # The Cartridge Header
 
-An internal information area is located at $0100-014F in each cartridge.
-It contains the following values:
+Each cartridge contains a header, located at the addresses `$0100`-`$014F`.
+The cartridge header contains the following information:
 
-## 0100-0103 - Entry Point
+## 0100-0103 – Entry Point
 
-After displaying the Nintendo Logo, the built-in boot procedure jumps to
-this address ($0100), which should then jump to the actual main program
-in the cartridge. Usually this 4 byte area contains a `nop` instruction,
-followed by a `jp $0150` instruction. But not always.
+After displaying the Nintendo logo, the built-in boot procedure jumps to
+this address (`$0100`), which should then jump to the actual main program
+in the cartridge. Usually (but not always) this 4-byte area contains a `nop`
+instruction followed by a `jp $0150` instruction.
 
-## 0104-0133 - Nintendo Logo
+## 0104-0133 – Nintendo Logo
+
+These bytes contain a bitmap of the Nintendo logo that is displayed
+when the Game Boy is powered on. The hex dump of this bitmap is:
 
-These bytes define the bitmap of the Nintendo logo that is displayed
-when the Game Boy gets turned on. The hex dump of this bitmap is:
 ```
- CE ED 66 66 CC 0D 00 0B 03 73 00 83 00 0C 00 0D
- 00 08 11 1F 88 89 00 0E DC CC 6E E6 DD DD D9 99
- BB BB 67 63 6E 0E EC CC DD DC 99 9F BB B9 33 3E
+CE ED 66 66 CC 0D 00 0B 03 73 00 83 00 0C 00 0D
+00 08 11 1F 88 89 00 0E DC CC 6E E6 DD DD D9 99
+BB BB 67 63 6E 0E EC CC DD DC 99 9F BB B9 33 3E
 ```
-The Game Boy's boot procedure verifies the content of this bitmap
-(after it has displayed it), and LOCKS ITSELF UP if these bytes are
-incorrect. A CGB verifies only the first half ($18 bytes of) the bitmap, but
-others (for example a Game Boy pocket) verify all $30 bytes, as does the
+
+The Game Boy's boot procedure **first displays** the logo and **then verifies**
+its contents. If the contents do not match the values above, the boot
+ROM **locks itself up**.
+
+A CGB verifies only the first half (`$18` bytes) of the bitmap, but other models
+(the Game Boy Pocket, for example) verify all `$30` bytes, as does the
 Game Boy Advance.
```

Comment:
This part is quite unclear. Other models (like game pocket) verify all bytes, like the GBA? Doesn't sound okay

Reply:
The GBA does not check all bytes actually (which can be seen in the disassembly https://github.com/ISSOtm/gb-bootroms/blob/master/src/cgb.asm).

Reply:
Thanks for the heads up @CasualPokePlayer. Link is a bit problematic as the source is leaked material. Maybe we can just link to a specific line or give some context and a link instead of copying that information here?

Reply:
https://www.youtube.com/watch?v=0dJrYeRbIgc there's also this video showing the GBA only doing the half logo check.

Reply:
> https://www.youtube.com/watch?v=0dJrYeRbIgc there's also this video showing the GBA only doing the half logo check.

This may be good enough for now. Maybe we can prepare a test rom?

Reply:
Better wording here might just be "CGB and future devices only check the first half, while all pre-CGB devices check the full logo." Although an interesting point in that is on the SGB, while the full logo is checked, it is not be the bootrom. The bootrom simply sends the logo to the SNES software (as can be seen in the diassembly in that repo), which then that SNES software verifies the logo (and refuses to boot if it's wrong).